src/versions/saveVersion.ts

```diff
@@ -101,12 +101,12 @@ export const saveVersion = async ({
 
   if (max > 0) {
     await enforceMaxVersions({
-      id,
       payload,
-      Model: VersionModel,
-      slug: entityConfig.slug,
-      entityType,
+      VersionModel,
```

Comment:
The property that needs to be passed to the `enforceMaxVersions` args object is called `Model`. I see that you're passing it a key of `VersionsModel`, which will break things